modules/aws-config/main.tf

```diff
@@ -40,11 +40,32 @@ module "utils" {
   context = module.this.context
 }
 
+locals {
+  account_enabled = local.enabled && var.scope == "account"
+  org_enabled     = local.enabled && var.scope == "organization"
+}
+
```

Comment:
Doesn't this approach prevent us from mixing account conformance packs from organizational ones? What if we only want certain conformance packs in PHI accounts, for example, but want others for the whole org?

I think it would be better to add a flag `var.conformance_packs` to indicate if it's an org conformance pack and default to account if not specified.

Reply:
almost done with this change. Testing it locally

Reply:
I found that setting a `default_scope` should be supported. @mcalhoun, I'll resolve this for now. I investigated the pack yaml's like you mentioned, and it looks like they are able to handle either scenario. I don't see any regressions with my changes. If you'd like to test this internally and make sure, I'm happy to help and to also show my own work (I've internally tested against a live aws-config component)